.github/workflows/ci.yml

```diff
@@ -31,3 +31,36 @@ jobs:
 
   vitest:
     uses: ./.github/workflows/vitest.yml
+
+  release:
+    needs: vitest
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/main' }}
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: npm
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      - run: npm run build
+      - name: Establish version
```

Comment:
Is it worth adding to the commit message that this code is copied from https://github.com/alphagov/paste-html-to-govspeak/blob/975879e4630d90625f958f4993b9508014825db1/.github/workflows/ci.yml#L54-L70?